standalone-metastore/metastore-server/src/test/java/org/apache/hadoop/hive/metastore/dbinstall/rules/Mssql.java

```diff
@@ -63,7 +64,8 @@ public String getJdbcDriver() {
 
   @Override
   public String getJdbcUrl(String hostAddress) {
-    return "jdbc:sqlserver://" + hostAddress + ":1433;DatabaseName=" + HIVE_DB + ";";
+    //Mssql doesn't support database parameter in docker
```

Comment:
I don't understand what the comment means. Can you elaborate more?

Reply:
The Mssql docker init doesn’t have database parameter for initialisation like postgres/mysql etc. so the initial jdbc connection should go to default url if the database was created with docker database argument.(https://github.com/microsoft/mssql-docker/issues/2)